providers/src/airflow/providers/google/CHANGELOG.rst

```diff
@@ -27,6 +27,152 @@
 Changelog
 ---------
 
+11.0.0
+......
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+.. warning::
+  All deprecated classes, parameters and features have been removed from the Google provider package.
+  The following breaking changes were introduced:
```

Comment:
:heart: the longlist of changes here :)

Reply:
Sorry, here are not all deprecations removed. Only those that reached their EOL.

Reply:
I updated the changelog.